experimental/generation/generator/test/commands/dialog/forms/unittests1.schema

```diff
@@ -0,0 +1,117 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "properties": {
+        "Quantity": {
+            "type": "number",
+            "minimum": 1,
+            "maximum": 10,
+            "$entities": [
+                "number"
+            ]
+        },
+        "Length": {
+            "type": "integer",
+            "minimum": 11,
+            "maximum": 20,
+            "$entities": [
+                "integer"
+            ]
+        },
+        "Complete": {
+            "type": "boolean",
+            "$entities": [
+                "boolean"
+            ]
+        },
+        "NameArray": {
+            "type": "array",
+            "items": {
+                "type": "string"
+            },
+            "$entities": [
+                "personName"
+            ]
+        },
+        "Email": {
+            "type": "string",
+            "format": "email"
+        },
+        "URL1": {
+            "type": "string",
+            "format": "uri"
+        },
+        "URL2": {
+            "type": "string",
+            "format": "iri"
+        },
+        "Name": {
+            "type": "string",
+            "$entities": [
+                "personName"
+            ]
+        },
+        "PhoneNumber": {
+            "type": "string",
+            "$entities": [
+                "phonenumber"
+            ]
+        },
+        "KeyphraseProperty": {
+            "type": "string",
+            "$entities": [
+                "keyPhrase"
+            ]
+        },
+        "Percentage": {
+            "type": "string",
+            "$entities": [
+                "percentage"
+            ]
+        },
+        "Age": {
+            "$ref": "template:age.schema#/age"
+        },
+        "OrdinalV2": {
+            "$ref": "template:ordinal.schema#/ordinalV2"
+        },
+        "GeographyV2": {
+            "$ref": "template:geography.schema#/geographyV2"
+        },
+        "Money": {
+            "$ref": "template:money.schema#/money"
+        },
+        "Temperature": {
+            "$ref": "template:temperature.schema#/temperature"
+        },
+        "Dimension": {
+            "$ref": "template:dimension.schema#/dimension"
+        },
+        "Datetime": {
+            "$ref": "template:datetime.schema#/datetime"
+        }
```

Comment:
We should also eventually have just bare date, time and date-time format which should connect up to the corresponding templates date.lg, time.lg and date-time.lg.  In the templates for that we need to ensure that the underlying entities are mapped into the surface form defined by the template, i.e. iso8601.  That is contrast to this form which we should also support where you get back a timex expression that can be manipulated.